### PR TITLE
Fix proxy configuration not getting set

### PR DESCRIPTION
**Pull request by MikeDevresse**

### CODE REVIEW

**Review by DonutsNL — APPROVED** [review]
Thanks for addressing allot of styling issues as well. You look like an experienced coder. Care to help me write the new version as well?

**Comment by DonutsNL** [comment]
Before i merge i also want to check if the proxy setting isnt handled in the /inc/acs.class.php as well.

File: inc/acs.class.php [hunk 1]
```diff
@@ -24,30 +24,32 @@
...
+}
 
 class PluginPhpsamlAcs
 {
     private const DS = DIRECTORY_SEPARATOR;
 
-    private $pathInfo        = [];
-    private $samlResponse    = null;
-    private $phpsaml         = null;
+    private $pathInfo = [];
```
[the remark this answers is not in the document]
Comment (MikeDevresse) [reply inc/acs.class.php:48]: my bad the ide changed everything i will push again with everything correct

**Comment by MikeDevresse** [comment]
Pr didn't change much tbh, would love to help you unfortunately it's pretty hard between personal and work life at this time.

**Review by DonutsNL — APPROVED** [review]

**Comment by DonutsNL** [comment]
I hear ye. Well consider it an standing invite then 👍

**Pull request merged**